ports/servoshell/desktop/minibrowser.rs

```diff
@@ -366,6 +366,7 @@ impl Minibrowser {
                             if let Some(event) =
                                 Self::browser_tab(ui, label, webview.focused, webview_id)
                             {
+                                location_dirty.set(false);
```

Comment:
While this fixes the reported issue, I find this area a bit buggy. For example, now with below steps url is empty when we open new tab.

- open a new tab
- delete `servo:newtab`
- open new tab / `cmd + T`
- url is empty
- go to previous tab and then go back to new tab again url is not empty anymore

Reply:
Indeed. Would be nice to find a better solution but for now i think it's fine to land this